src/Clustering/GMUClusterManager.h

```diff
@@ -45,13 +45,16 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @interface GMUClusterManager : NSObject<GMSMapViewDelegate>
 
+/** Disallow default initializer */
+- (id)init __attribute__((unavailable("Use initWithMap:algorithm:renderer: instead.")));
```

Comment:
I think we'd like to use a macro like NS_UNAVAILABLE. Can you please find out from which SDK version was the macro introduced?